.github/workflows/gem-push.yml

```diff
@@ -0,0 +1,31 @@
+name: Ruby Gem
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    name: Build + Publish
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.3.x
```

Comment:
2.3 or 2.6 as name says?

Reply:
2.3 is what we want, the name was wrong (copy/paste error)